game/src/core/utils/components/draw/CoreAnimationPathEnum.java

```diff
@@ -0,0 +1,16 @@
+package core.utils.components.draw;
+
+/**
+ * This enum stores the default paths to the animations used by the systems inside the core package.
```

Comment:
Irgendwo sollte der Zusammenhang zw. Assets-Ordner, dem Pfad für `DrawComponent` und den Pfaden hier in den Konstanten genauer erklärt werden. Javadoc oder eher Doku? Falls ersteres, wo? Falls letzteres, bitte hier dorthin verlinken.

Reply:
Ich glaub das ist schon was für die javadoc.
Ich bin selbst nocht nicht so richtig glücklich damit. 
Strukturell würde ich das `DrawComponent` als zentralen Startpunkt sehen und daher dort am meisten erklären und auf die anderen Sachen verlinken. 
Als User guck ich nicht in irgendeine Hilfsklasse rein sondern geh direkt zum Component.